### PR TITLE
Test that client rejects an Update Control map which is broken.

### PR DESCRIPTION
**Pull request by kacf**
I forgot to submit this one earlier.

### CODE REVIEW

**Comment by kacf** [comment]
@mender-test-bot start pipeline

File: tests/tests/test_update_control.py [hunk 1]
```diff
@@ -127,3 +127,67 @@ def test_update_control(
...
+        devauth.accept_devices(1)
+
+        deploy = Deployments(auth, devauth)
+
+        devices = list(
+            set([device["id"] for device in devauth.get_devices_status("accepted")])
+        )
+        assert 1 == len(devices)
+
+        wait_for_connect(auth, devices[0])
```
Comment (tranchitella) [inline tests/tests/test_update_control.py:162]: Out of curiosity, why do you need to wait for connect?

**Review by tranchitella — APPROVED** [review]

File: tests/tests/test_update_control.py (hunk 1 above)
Comment (kacf) [reply tests/tests/test_update_control.py:162]: I don't know, actually. All this boiler plate code was copied from the previous test, which @oleorhagen wrote. I wouldn't be surprised if this was again copied from somewhere else that actually needs it. I'll merge this as is, but I can try removing them in a followup.

Comment (kacf) [reply tests/tests/test_update_control.py:162]: https://github.com/mendersoftware/integration/pull/1460

**Pull request merged**